lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -437,14 +437,24 @@ class ByteCodeBufferBuilder
             V3.value = buildDateHash;
             V4.value = buildTimeHash;
         }
-        else
+        else if (!GenerateLibraryByteCode())
         {
             auto guidDWORDs = (DWORD*)(&byteCodeCacheReleaseFileVersion);
             V1.value = guidDWORDs[0];
             V2.value = guidDWORDs[1];
             V3.value = guidDWORDs[2];
             V4.value = guidDWORDs[3];
         }
+        else
+        {
+            // To keep consistent library code between Chakra.dll and ChakraCore.dll, use a fixed version.
+            // This goes hand in hand with the bytecode verification unit tests.
+            V1.value = 0;
+            V2.value = 0;
+            V3.value = 0;
+            V4.value = 0;
```

Comment:
Wait, I understand now. For library code, we don't want to do the GUID check? We still need to make sure the ByteCode is updated when it is supposed to be. How else can we specifically test this?

---
In reply to: [76506572](https://github.com/Microsoft/ChakraCore/pull/1505#discussion_r76506572) [](ancestors = 76506572)